src/js/node/tty.ts

```diff
@@ -1,81 +1,74 @@
+// Hardcoded module "node:tty"
+
 // Note: please keep this module's loading constrants light, as some users
 // import it just to call `isatty`. In that case, `node:stream` is not needed.
+
 const {
   setRawMode: ttySetMode,
   isatty,
   getWindowSize: _getWindowSize,
 } = $cpp("ProcessBindingTTYWrap.cpp", "createBunTTYFunctions");
 
 const { validateInteger } = require("internal/validators");
+const fs = require("node:fs");
 
 function ReadStream(fd): void {
   if (!(this instanceof ReadStream)) {
     return new ReadStream(fd);
   }
-  require("node:fs").ReadStream.$apply(this, ["", { fd }]);
+  fs.ReadStream.$apply(this, ["", { fd }]);
```

Comment:
Can we keep this lazy?

Reply:
`$toClass(ReadStream, "ReadStream", require("node:fs").ReadStream);` is needed for `test/js/node/test/parallel/test-net-access-byteswritten.js` so unlikely. we can likely make `node:fs` load faster